apps/dashboard/app/(app)/apis/[apiId]/settings/actions.ts

```diff
@@ -1,7 +1,11 @@
 "use server";
 
-import { revalidatePath } from "next/cache";
+import { revalidatePath, revalidateTag } from "next/cache";
 
 export async function revalidate() {
   await revalidatePath("/", "layout");
 }
+
+export async function revalidateMyTag(tag: string) {
```

Comment:
what's the purpose of this function?
why don't we just import `revalidateTag` from `next/cache` in our components?

Reply:
we can't use revalidateTag in client side, we are creating wrapper around to use it in client side !

Reply:
aah gotcha, makes sense

but then let's reexport it as the same name, `revalidateMyTag` looks weird

Reply:
"Agreed, a typesafe wrapper for cache tags would help prevent typos and make debugging easier.

Reply:
"Should we make any other changes or does everything look fine to you?"


> aah gotcha, makes sense
> 
> but then let's reexport it as the same name, `revalidateMyTag` looks weird

Ok ! "Should we make any other changes or does everything look fine to you?"